Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ConfigChangeDetector.kt

```diff
@@ -46,8 +47,17 @@ class ConfigChangeDetector @Inject constructor(
         val oldConfigId = riskLevelSettings.lastUsedConfigIdentifier
         if (newIdentifier != oldConfigId) {
             Timber.tag(TAG).i("New config id ($newIdentifier) differs from last one ($oldConfigId), resetting.")
-            riskLevelStorage.clear()
-            taskController.submit(DefaultTaskRequest(RiskLevelTask::class, originTag = "ConfigChangeDetector"))
+            riskLevelStorage.clearResults()
+            taskController.submit(
+                DefaultTaskRequest(RiskLevelTask::class, originTag = "ConfigChangeDetector")
+            )
+            taskController.submit(
+                DefaultTaskRequest(
+                    PresenceTracingWarningTask::class,
+                    arguments = PresenceTracingWarningTask.Arguments(true),
```

Comment:
We need to add the arguments in other places with 

```
            taskController.submit(
                DefaultTaskRequest(
                    PresenceTracingWarningTask::class,
```

too, otherwise the `arguments as Arguments` will through a cast exception.

Reply:
You were right and the arguments may not be necessary, so my comment is obsolete once refactored.